client/my-sites/plans-v2/product-card/index.tsx

```diff
@@ -143,9 +148,7 @@ const ProductCardWrapper = ( {
 			isOwned={ isOwned }
 			isDeprecated={ item.legacy }
 			className={ className }
-			UpgradeNudge={
-				<UpgradeNudgeWrapper item={ item } currencyCode={ currencyCode } onClick={ onClick } />
-			}
+			UpgradeNudge={ UpgradeNudge }
```

Comment:
I feel it would be cleaner for the to code pass in `isOwed` as a prop and let the component return `null` if it's `false`.

Reply:
I thought about that option as well, but it didn't felt right to me to pass down that prop when the component doesn't really care about it. `isOwned` doesn't affect the state or UI of the UpgradeNudge in any way and by passing down the prop we force React to run the component's function/code even when we know that if `isOwned` is false we should not render it.

I agree that it seems to add a bit of noise to the parent component, but if you look at this from the `UpgradeNudge` consumer perspective, why would you need to pass down that prop to hide or show the component? That seems like a concern of the parent component, unless showing/hiding involves a piece of state that lives inside the child component.